### PR TITLE
made sidebar button active

### PR DESCRIPTION
**Pull request by binayakbehera999**
Signed-off-by: Binayak Behera <binayakbehera999@gmail.com>
added active class to sidebar buttons

### CODE REVIEW

File: src/components/layouts/Sidebar.tsx [hunk 1]
```diff
@@ -3,14 +3,27 @@ import "./layouts.style.css";
 import { HashRouter as Router, Link } from "react-router-dom";
 
 export default class Sidebar extends React.Component<{}> {
+  state = {
+    active: Array(4).fill(false)
+  };
+
+  activeClassToggle = x => {
+    var activeState = this.state.active;
```
Comment (ankitjena) [inline src/components/layouts/Sidebar.tsx]: Don't use var

File: src/components/layouts/Sidebar.tsx [hunk 2]
```diff
@@ -3,14 +3,27 @@ import "./layouts.style.css";
...
+    activeState.fill(false);
+    activeState[x] = true;
+    this.setState({ active: activeState });
```
Comment (ankitjena) [inline src/components/layouts/Sidebar.tsx]: Use callback as suggested by deepscan

**Comment by ankitjena** [comment]
It's better if we start using `antd` from this point. Active styling in this case will be provided by default.

File: src/components/layouts/Sidebar.tsx [hunk 3]
```diff
@@ -45,19 +68,24 @@ export default class Sidebar extends React.Component<{}> {
...
+              <Link to="/settings" style={{ textDecoration: "none" }}>
+                <div>
+                  <div
+                    className={
+                      this.state.active[3] ? "sidebar-button-active" : ""
+                    }
+                    onClick={e => this.activeClassToggle(3)}
+                  >
+                    <img
+                      src="assets/icons/settings-icon.svg"
```
Comment (ganeshpatro321) [inline src/components/layouts/Sidebar.tsx]: Fix the alignment here

Setting button

Screenshot from 2019-10-25 15-53-12

**Comment by Harkishen-Singh** [comment]
@binayakbehera999 any updates?

**Comment by binayakbehera999** [comment]
There a some issues while using ant design, need to resolve that first

**Comment by Harkishen-Singh** [comment]
@binayakbehera999 don't use ant design as of now. we cannot afford any increase in size.

**Comment by Harkishen-Singh** [comment]
@binayakbehera999 any updates?

**Comment by ganeshpatro321** [comment]
@binayakbehera999 status?

**Comment by Harkishen-Singh** [comment]
are we closing this?

**Pull request closed without merging**